src/daemon/rpc_command_executor.cpp

```diff
@@ -2768,42 +2762,58 @@ bool rpc_command_executor::prepare_eth_registration(std::string_view operator_ad
     auto ed_sig = reg_info["service_node_signature"].get<std::string>();
     auto bls_pubkey = reg_info["bls_pubkey"].get<std::string>();
     auto bls_sig = reg_info["proof_of_possession"].get<std::string>();
-    if (print) {
-        tools::msg_writer("Printing info for L2 staking\n");
+    if (url.empty()) {
         tools::msg_writer(
-                "ed25519_pubkey: {}\n"
-                "bls_pubkey: {}\n"
-                "ed25519_signature: {}\n"
-                "bls_signature: {}\n"
-                "operator_address: {}\n",
-                snode_pubkey,
-                bls_pubkey,
-                ed_sig,
-                bls_sig,
-                operator_address);
-    } else {
+                "{}\n",
+                fmt::styled("L2 Contract Registration Information:", fmt::emphasis::underline));
         tools::msg_writer(
-                "Submitting L2 staking information to staking.getsession.org, please wait.");
-        // TODO: make config option
-        auto url = cpr::Url{"https://ssb.oxen.observer/api/store/{}"_format(snode_pubkey)};
+                "{} {}\n"
+                "{} {}{}\n"
+                "{} {}{}\n"
+                "{} {}{}{}{}\n"
```

Comment:
I know there's colours here but potentially adding a space between the 32 byte chunks is more useful because it means when you double click on the word-chunk the word-selection will be delimited by the space and select just the 32 byte chunk instead of the entire word.

This makes the piece-meal copy and paste into the 32 byte input text fields on the blockchain explorers easier than having to drag the cursor over the word and delimit the selection yourself based off the colour of the chunk.

Reply:
Good idea; updated.